schema.xml

```diff
@@ -570,6 +570,7 @@
         is added to the index.  It's used either to index the same field differently,
         or to add multiple fields to the same field for easier/faster searching.  -->
    <!-- Copy Fields -->
+   <copyField source="lc_call_number_display" dest="lc_call_number_t"/>
```

Comment:
Just curious about this... Do we need lc_call_number_t for sorting purposes?  And if so, don't we already have some specialized fields that @quoideneuf added that have been normalized so that they actually sort correctly?

Reply:
I was doing this so that the field would be indexed.

Reply:
I mean why do we need the index, for sorting?

Reply:
Because I believe we are already indexing a normalized version.

Reply:
OK @quoideneuf reports that what he's indexing is not the full field.